src/custom/theme/index.tsx

```diff
@@ -17,6 +22,30 @@ export enum Z_INDEX {
   tooltip = 1080,
 }
 
+type TextProps = TextPropsUni & { override?: boolean }
+
+export const TextWrapper = styled(Text)<{ color: keyof Colors; override?: boolean }>`
+  color: ${({ color, theme, override }) => {
+    const colour = (theme as any)[color]
+    if (colour && override) {
+      return colour + '!important'
```

Comment:
was an approach I took to use sparingly and in-case of CSS descendancy clashing as an escape-hatch. use `override` to force `!important` and render the correct colour